src/evaluation_system/model/file.py

```diff
@@ -98,7 +98,7 @@ def from_dict(cls, dataset: str, drs_dict: dict[str, Any]) -> DRSStructure:
             DRSStructure created from dict
         """
         d = cls(
-            root_dir=drs_dict["root_dir"],
+            root_dir=drs_dict.get("root_path", drs_dict.get("root_dir")),
```

Comment:
i'm not against of this approach but i think it would be better to give the user/freva admin somewhere a log to make the key changes for the future ...